package.json

```diff
@@ -44,7 +45,11 @@
     "android": "(cd ./native && npm run android)",
     "check-owners": "npm whoami && grep -E $(npm whoami) NPM_OWNERS && node scripts/npm/check-owners.js",
     "publish": "npm run check-owners && npm run build && git pull --rebase && npm run test && lerna publish",
-    "danger": "danger"
+    "danger": "danger",
+    "prettier": "prettier --config .prettierrc --write \"{packages,native,scripts}/**/*.js\" dangerfile.js webpack.config.babel.js"
+  },
+  "lint-staged": {
+    "*.js": ["eslint"]
```

Comment:
How does this relate to our `npm run lint:js`script? I.e. we have some custom stuff in there (`eslint . .storybook --ext .js,.jsx`). It feels like its going down a different path.

Reply:
Shouldn't be a problem. `--ext` is covered by the `*.js` filter and the other args are which files to lint and we don't want to explicitly lint everything here

Reply:
We don't have any jsx files as far as I can see